packages/taquito-sapling/src/constants.ts

```diff
@@ -1,2 +1,4 @@
 export const KDF_KEY = 'KDFSaplingForTezosV1';
 export const OCK_KEY = 'OCK_keystringderivation_TEZOS';
+export const defaultMemo = '';
```

Comment:
kind of a nitpick, but for constants maybe we should keep the same format like `DEFAULT_MEMO`

Reply:
Will address